crates/ra_hir/src/ids.rs

```diff
@@ -74,7 +74,7 @@ impl HirFileId {
         }
     }
 
-    pub(crate) fn as_original_file(self) -> FileId {
+    pub fn as_original_file(self) -> FileId {
```

Comment:
This was originally "private" because there shouldn't be such function at all :) 

Instead, we should always use `orignial_file`, which should also return a map translating offsets between the two files. 

That however seems not too close, so it's OK to expose this. However maybe it makes sense to add a comment here?

```
// XXX: this is a temporary function, which should go away when we implement namresolution+macro expansion combo. Prefer using `original_file` if possible.
```